Podfile

```diff
@@ -25,6 +25,7 @@ target 'Susi' do
   pod 'ReachabilitySwift'
   pod 'Localize-Swift'
   pod 'paper-onboarding'
+  pod 'NotificationBannerSwift'
```

Comment:
Can you please search for something alternative to this? This pod is also required `MarqueeLabel` and `SnapKit` which is increasing the size of the project.

Reply:
@jogendra , I searched for the other alternative , and found that others libraries are not having status bar banner , they have only basic banners and the problem with basic banners is that it overlaps the Susi skill button in chatVC , when comes into an existence . Do you know any alternative to this or can we go for basic banner?